extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -0,0 +1,46 @@
+import {Layout_Enum} from '#core/dom/layout';
+
+import {Services} from '#service';
+
+import {CSS} from '../../../build/amp-story-audio-sticker-0.1.css';
+import {
+  Action,
+  StateProperty,
+} from '../../amp-story/1.0/amp-story-store-service';
+
+const TAG = 'amp-story-audio-sticker';
+
+export class AmpStoryAudioSticker extends AMP.BaseElement {
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {?../../../extensions/amp-story/1.0/amp-story-store-service.AmpStoryStoreService} */
+    this.storeService_ = null;
+  }
+
+  /** @override */
+  buildCallback() {
+    return Services.storyStoreServiceForOrNull(this.win).then(
+      (storeService) => {
+        this.storeService_ = storeService;
+
+        this.element.addEventListener('click', (event) => {
+          if (this.storeService_.get(StateProperty.MUTED_STATE)) {
+            this.storeService_.dispatch(Action.TOGGLE_MUTED, false);
+            event.stopPropagation();
+          }
```

Comment:
You don't need to check the state, right? If you always toggle to `false` then it will just not update the store if it's already false.

It might make more sense to just do this:

```css
amp-story:not([muted]) amp-story-audio-sticker {
  pointer-events: auto !important;
}
amp-story-audio-sticker * {
  pointer-events: none !important;
}
```

And the clicks won't fire in the other scenarios. First check if we need the second rule in case pubs want to make the contents clickable anyways.